pycue/opencue/api.py

```diff
@@ -79,7 +79,9 @@ def getDefaultServices():
     """
     Return the default service list.  Services
     define the default application features.
-    @rtype list<Service>
+
+    :rtype: list
```

Comment:
For anyone using PyCharm, they can set Tools > Python Integrated Tools > Docstrings > Docstring format to "reStructuredText". This will let you use standard PyCharm tools for generating docstrings in this format.

Reply:
(mostly leaving this as a note for myself)

Reply:
Thanks, I've added an issue to remind us to update the contributor docs with this tip: https://github.com/AcademySoftwareFoundation/opencue.io/issues/143